aries_cloudagent/anoncreds/registry.py

```diff
@@ -147,6 +147,15 @@ async def get_revocation_list(
         resolver = await self._resolver_for_identifier(rev_reg_def_id)
         return await resolver.get_revocation_list(profile, rev_reg_def_id, timestamp)
 
+    async def get_revocation_registry_delta(
+        self, profile: Profile, rev_reg_def_id: str, timestamp: None
+    ) -> Tuple[dict, int]:
+        """Get a revocation registry delta."""
+        resolver = await self._resolver_for_identifier(rev_reg_def_id)
+        return await resolver.get_revocation_registry_delta(
+            profile, rev_reg_def_id, timestamp
+        )
+
```

Comment:
Looks like this is the method ultimately calling the method I tagged in my last comment and that this is called from revocation routes. Since the deltas are specific to Indy, I recommend having the revocation routes grab an instance of the `LegacyIndyRegistry` directly and call the `get_revocation_registry_delta` method on it.

Reply:
OK, thanks for this. I have refactored base and moved the logic directly into routes. The same approach was used to implement the endpoints used in the TAA BDD tests (see #2493).